packages/twenty-server/src/engine/seeder/data-seeds/pets-data-seeds.ts

```diff
@@ -0,0 +1,2852 @@
+export const PETS_DATA_SEEDS = [
```

Comment:
I think we should add large files like this in ignorePatterns of our eslint config. Check eslintrc.cjs we should already have some

Reply:
Ok